$OpenBSD$

Setting file permission on config.xml to allow user writing.

--- src/RadioTray.py.orig	Thu Nov 20 23:37:33 2014
+++ src/RadioTray.py	Thu Nov 20 23:38:04 2014
@@ -162,6 +162,7 @@ class RadioTray(object):
 
             self.logger.warn('Configuration file not found. Copying default configuration file to user directory')
             copy2(DEFAULT_CONFIG_FILE, self.cfg_filename)
+            os.chmod(self.cfg_filename, 0644)
 
 
     def configLogging(self):
